packages/preview/showman/0.1.1/typst.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "showman"
+description = "Eval & show typst code outputs inline with their source"
+repository = "https://github.com/ntjess/showman"
+keywords = ["manual", "documentation", "eval", "inline"]
+exclude = ["manual.pdf"]
+version = "0.1.1"
+entrypoint = "lib.typ"
+authors = ["Nathan Jessurun"]
+license = "Unlicense"
+compiler = "0.11.0"
```

Comment:
May I suggest adding the `categories` key to the manifest? This will make the package show up in the respective category on https://typst.app/universe. You can find [a list of the categores here](https://github.com/typst/packages/blob/main/CATEGORIES.md).